setup.py

```diff
@@ -360,6 +360,12 @@ def setup_package():
         cmdclass={"sdist": sdist_checked},
         python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*',
         zip_safe=False,
+        entry_points={
+            'console_scripts': [
+                'f2py = numpy.f2py.__main__:main',
+                'conv-template = numpy.distutils.conv_template:main',
```

Comment:
`conv-template` or `conv_template`?

Reply:
There are other scripts, should they be added also?

Reply:
1. Command names usually contain dashes.
2. Which other scripts are there?

Reply:
There is `from_template.py`, used by f2py and `cythonize.py`. I'm not clear just what needs an entry point or why. Note that `process_file` is the part of `conv_template` that is actually imported and used in the build, `conv_template` is used as a module rather than a script.